### PR TITLE
Use bundler API to get Gemfile dependencies

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: get-gemfile-deps [hunk 1]
```diff
@@ -6,59 +6,54 @@ filename = ARGV.shift || 'Gemfile'
...
+      target = requires[group][req]
+      name = package(dependency.name)
+
+      dependency.requirement.requirements.each do |op, version|
+        if op == '~>'
+          target << "#{name} >= #{version}"
+          target << "#{name} < #{version.bump}.0"
+        elsif op == '>=' && version.to_s == '0'
+          target << name
+        elsif op != '!=' # != is not supported in RPM
```
Comment (ekohl) [inline get-gemfile-deps:47]: The triple use of `!=` was not really intentional

Comment (evgeni) [reply get-gemfile-deps:47]: That's the Rammstein line ;)

Comment (evgeni) [reply get-gemfile-deps:47]: That's the Rammstein line.

**Comment by ekohl** [comment]
Test failed because there was no change to the release, but I don't think this really needs a release of Foreman/Foreman Proxy.

**Comment by ekohl** [comment]
@evgeni anything left here? The build failed, but that's unrelated.

**Comment by evgeni** [comment]
I didn't look yet, will do tomorrow

**Review by evgeni — APPROVED** [review]

**Pull request merged**